lib/rmt/cli/export.rb

```diff
@@ -1,7 +1,7 @@
 class RMT::CLI::Export < RMT::CLI::Base
 
-  desc 'data PATH', 'Store SCC data in files at given path'
-  def data(path)
+  desc 'scc-data PATH', 'Store SCC data in files at given path'
```

Comment:
What was the thinking in renaming this 'scc-data'? It seems overly dependent on the identity of scc, which might change over the lifespan of rmt. 'upstream-data' instead?